enacts/wat_bal/layout_monit.py

```diff
@@ -309,7 +358,67 @@ def controls_layout(lat_min, lat_max, lon_min, lon_max, lat_label, lon_label):
                         Sentence(
                             Number("kc_end", CONFIG["kc_v"][4], min=0, max=2, html_size=4),
                         ),
-                        dbc.Button(id="submit_kc", children='Submit'),
+                        dbc.Button(
+                            id="submit_kc",
+                            children='Submit',
+                            color="light",
+                            style={"color": "green", "border-color": "green"},
+                        ),
+                        border_color="green",
+                    ),
+                    Block(
+                        "Compare to...",
+                        Sentence(
+                            "Planting Date",
+                            DateNoYear("planting2_", 1, CONFIG["planting_month"]),
+                            "",
+                            Number(
+                                "planting2_year",
+                                other_year_default,
+                                min=other_year_min,
+                                max=other_year_max,
+                                html_size=5
+                            ),
+                        ),
+                        Sentence(
+                            "for",
+                            Text("crop2_name", CONFIG["crop_name"]),
+                            "crop cultivars: initiated at",
```

Comment:
This and the corresponding input in the previous Block should be a Select (choose from a predefined set of options) rather than an unrestricted Text input?

Reply:
Not at this point. The crop name text is mostly for cosmetics. There may be another version alter when the user would actually pick a crop name in a list, and that would set Kc values (and inform the user what those values are). But in this version, the user picks the "crop" by setting the Kc values themselves, and they can give it a name for convenience, cosmetics.

Reply:
Ah. I tried typing in "ffgg" as the crop name, and when it didn't result in an error message, I concluded that something was wrong with the error handling. It never occurred to me that was meant to be an unconstrained free-text field.

I'll allow that I'm not typical of the users of this application, but I find this confusing/surprising.

Reply:
I'll come up with something else in the next PR.